problems/5.loops.js

```diff
@@ -1,33 +1,48 @@
 /** Returns every even number from 0 to a given number
-* @param {number} num - The number to iterate up to
-* @param {number[]} - An array of even numbers from 0 to {num}
-*/
-function evenNumbers() {
+ * @param {number} num - The number to iterate up to
+ * @param {number[]} - An array of even numbers from 0 to {num}
+ */
+function evenNumbers(array) {
+    let newArray = [];
+    let arr = Math.floor(array.lenght % 2 === 0);
+    for (let i = 0; i < array.length; i++) {
+        if (newArray === arr)
+            newArray.push(arr)
+        return newArray
 
+    }
 }
 
 /** Returns all multiples of a given number up to 1000
  * Example if num was 3 the output would be [0, 3, 6, 9, 12, ..., 999]
-* @param {number} num - The number to find multiples of
-* @returns {number[]} - An array of all multiples of {num} up to 1000
-*/
-function multiplesToOneThousand() {
+ * @param {number} num - The number to find multiples of
+ * @returns {number[]} - An array of all multiples of {num} up to 1000
+ */
+function multiplesToOneThousand(arr) {
+    let newArray = []
+    let arr = Math.floor(1000 % 2 == 3);
+    for (i = o; i <= arr.length; i++)
```

Comment:
remember the {} syntax that you're missing.

Reply:
Ok. Thank you